gcp_variant_transforms/beam_io/vcf_parser.py

```diff
@@ -250,6 +254,7 @@ def __init__(self,
                compression_type,  # type: str
```

Comment:
Silly github won't allow me to comment above this section... - I think we should augment the doc to explain compressed files logic below.